views/localweb.html

```diff
@@ -0,0 +1,480 @@
+<!DOCTYPE html>
+<html lang="{{ localeInfo.lang }}" dir="{{ localeInfo.direction }}">
+<head>
+  <meta charset="utf-8">
+  {{ newrelic.getBrowserTimingHeader() | safe }}
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+  <meta name="csrf-token" content="{{ csrf }}">
```

Comment:
This is for login. csrf token is a unique id that connects you to a signed in user server side. If they match, it tells the server this user is legit. Ping Cade for more info.

If you don't have any login on this page, you can remove it.